lnrpc/invoicesrpc/invoices.proto

```diff
@@ -192,3 +200,60 @@ message LookupInvoiceMsg {
 
     LookupModifier lookup_modifier = 4;
 }
+
+// InvoiceAcceptorRequest is a message that the server sends to the client to
+// request the client to accept an invoice.
+message InvoiceAcceptorRequest {
+    // invoice is the invoice that the client should consider accepting.
+    lnrpc.Invoice invoice = 1;
```

Comment:
Cool, so this will have `InvoiceHTLC` which is really what the interceptor wants.

Reply:
we might wanna add the **wire level** custom records on `InvoiceHTLC` too, currently we only have
```
    // Custom tlv records.
    map<uint64, bytes> custom_records = 9;
```

Reply:
Yes. I think that's still needed. Not sure if in this PR.